### ISSUE TITLE
Missing cluster name

### ISSUE THREAD

**Issue by dlabrecq** [project-koku/koku#1536]
## Background
We recently removed an implicit group by cluster from the API for project/node which means that we don't have a single cluster to return here.

## Proposed Solution
- We can convert to return an array of clusters. This way if the project exists in multiple clusters we can still deliver this information in a single row in the details page / API resultset.

- See https://docs.djangoproject.com/en/3.0/ref/contrib/postgres/aggregates/#arrayagg for Django use of Postgres specific array_agg method. We could use ArrayAgg to package up cluster ids and cluster aliases in our annotations and aggregates.

**Describe the bug**
As a user, I want to see the cluster associated with each Ocp and Ocp on AWS project. The UI had provided this information in the details page, via table expandable rows. 

Unfortunately, the cost reports no longer appear to provide the cluster. As a result, the cluster is undefined and the UI no longer shows cluster names for each project.

**To Reproduce**
1. Go to 'OpenShift on cloud details' or 'OpenShift details'
2. Click on 'Group cost by' project
3. Expand any table row
4. See missing cluster name

**Expected behavior**
We need the cluster name associated with each project

**Screenshots**
This is the API used in the OpenShift details:
/v1/reports/openshift/costs/?delta=cost&filter[limit]=10&filter[offset]=0&filter[resolution]=monthly&filter[time_scope_units]=month&filter[time_scope_value]=-1&group_by[project]=*&order_by[cost]=desc

![Screen Shot 2019-12-11 at 11 43 17 AM](https://user-images.githubusercontent.com/17481322/70641343-7ae0eb00-1c0b-11ea-9bd7-92ca43864621.png)

This is the API used in the OpenShift on cloud details:
/v1/reports/openshift/infrastructures/aws/costs/?delta=cost&filter[limit]=10&filter[offset]=0&filter[resolution]=monthly&filter[time_scope_units]=month&filter[time_scope_value]=-1&group_by[project]=*&order_by[cost]=desc

![Screen Shot 2019-12-11 at 11 45 13 AM](https://user-images.githubusercontent.com/17481322/70641495-b8457880-1c0b-11ea-863c-c801e7cdba9a.png)

**Comment by myersCody** [comment]
After a discussion with @adberglund & @dlabrecq we decided to put this issue on hold pending a discussion with Sergio.

**Comment by dlabrecq** [comment]
Please include @nlcwong as well. We may need to redesign the details page a bit.

**Comment by myersCody** [comment]
## Update
After the meeting we decided that the `cluster_id` & `cluster_alias` will be returned as an array.

**Comment by lcouzens** [comment]
Verified backend changes, waiting on final frontend array handling.
